src/main/java/javaClasses/homework_4/arevik_arakelyan/B/Work.java

```diff
@@ -0,0 +1,17 @@
+package javaClasses.homework_4.arevik_arakelyan.B;
+
+public class Work {
+    public static void main(String[] args) {
+        Developer developer = new Developer();
+        QualityEngineer qualityEngineer = new QualityEngineer();
+        developer.setNames("Petros");
+        developer.setLastName("Petrosyan");
+        developer.setTechstack("PHP");
+        qualityEngineer.setNames("Arevik");
+        qualityEngineer.setLastName("Arakelyan");
+        qualityEngineer.setType("MANUAL");
+        System.out.println(developer.doWork());
+        System.out.println(qualityEngineer.doWork());
```

Comment:
Please provide an example of call getCompany() here. Don't forget to treat with it just like with any other static method.

Reply:
done